app/views/articles/_sidebar_nav.html.erb

```diff
@@ -45,29 +45,50 @@
 </nav>
 
 <nav class="mb-6" aria-label="Secondary sidebar nav">
-  <% if user_signed_in? %>
+<nav class="mb-6" aria-label="Secondary sidebar nav">
+  <% if user_signed_in? && current_user.following_tags_count >= 1 %>
     <header class="p-2 pr-0 flex items-center justify-between">
       <h3 class="crayons-subtitle-3">My Tags</h3>
       <a href="/dashboard/following_tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Customize tag priority" title="Customize tag priority">
         <%= inline_svg_tag("cog.svg", aria: true, class: "crayons-icon") %>
       </a>
     </header>
     <div id="sidebar-nav-followed-tags" class="overflow-y-auto mb-2" style="max-height: 42vh;"></div>
-  <% else %>
-    <h3 class="crayons-subtitle-3 p-2">Popular Tags</h3>
-    <div id="sidebar-nav-default-tags" class="overflow-y-auto" style="max-height: 42vh">
-      <% Tag.where(supported: true).order(hotness_score: :desc).limit(30).pluck(:id, :name).each do |tag_array| %>
-        <div class="sidebar-nav-element" id="default-sidebar-element-<%= tag_array.second %>">
-          <a class="crayons-link crayons-link--block py-2 px-2" href="<%= tag_path(tag_array.second) %>">
-            #<%= tag_array.second %>
-          </a>
-          <a class="follow-action-button sidebar-nav-link-follow crayons-btn crayons-btn--s"
-            href="#" id="sidebar-nav-link-follow-<%= tag_array.second %>"
-            data-info='{"id":<%= tag_array.first %>,"className":"Tag"}'>
-            Follow
-          </a>
+    <% elsif user_signed_in? %>
+      <header class="p-2 pr-0 flex items-center justify-between">
+        <a href="/tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Tags Overview" title="Tags Overview">
```

Comment:
Lets drop these classes here, since they add some unnecessary padding and make things misaligned.
Also, shouldn't we use `<%= tags_path %>` instead of `/tags` ?
 
```suggestion
        <a href="<%= tags_path %>" aria-label="Tags Overview" title="Tags Overview">
```

Reply:
Good call! 😄